routes/compose.js

```diff
@@ -0,0 +1,30 @@
+const router = require("express").Router();
+
+const Blog = require("../backend_model/Blog_post")
+
+
+router.get("/compose", (req, res) => {
+    res.send("/")
+})
```

Comment:
Why? Are you just testing the `get` route?

Reply:
solved.